src/Sylius/Bundle/ResourceBundle/DependencyInjection/Configuration.php

```diff
@@ -167,6 +167,8 @@ private function addTranslationsSection(ArrayNodeDefinition $node)
                     ->children()
                         ->scalarNode('default_locale')->cannotBeEmpty()->end()
                         ->scalarNode('locale_provider')->defaultValue('sylius.translation.locale_provider.request')->cannotBeEmpty()->end()
+                        ->scalarNode('available_locales_provider')->defaultValue('sylius.translation.locales_provider.array')->cannotBeEmpty()->end()
+                        ->arrayNode('available_locales') ->prototype('scalar')->end()
```

Comment:
If you change locales_provider then avaiable_locales will be unused. They are set as parameter of ArrayLocalesProvider in translation.xml from ResourceBundle.